assets/user-profile/index.ts

```diff
@@ -3,22 +3,30 @@ import {initWebSocket} from 'websocket';
 import UserProfileApp from './components/UserProfileApp';
 import {initData, selectMenu, pushNotification} from './actions';
 import {store} from './store';
-import {WIRE_TOPIC_FOLDERS_UPDATED} from 'wire';
 
 if (window.profileData) {
     store.dispatch(initData(window.profileData));
 }
 
+/**
+ * This constant needs to be defined here because of script loading.
```

Comment:
> This constant needs to be defined here

"Here" as opposed to where? Keep in mind when you're writing comments that whoever will read it, will not know about the PR that added the comment. When possible, write it in a way that is understandable by having access only to current commit. Like "this constant needs to be define here, and not in /path/file.ts". If the PR is needed to understand the comment - link to the PR.